include/gsl/multi_span

```diff
@@ -446,19 +478,28 @@ namespace details
             return this->Base::totalSize() * arr[Dim] + last;
         }
 
-        size_type totalSize() const GSL_NOEXCEPT { return CurrentRange * this->Base::totalSize(); }
+        GSL_SUPPRESS(c.128) // NO-FORMAT: attribute // no pointers to BoundsRanges should be ever used
+        size_type totalSize() const noexcept
+        {
+            return CurrentRange * this->Base::totalSize();
+        }
 
-        size_type elementNum() const GSL_NOEXCEPT { return CurrentRange; }
+        GSL_SUPPRESS(c.128) // NO-FORMAT: attribute // no pointers to BoundsRanges should be ever used
+        size_type elementNum() const noexcept
+        {
+            return CurrentRange;
+        }
 
-        size_type elementNum(std::size_t dim) const GSL_NOEXCEPT
+        GSL_SUPPRESS(c.128) // NO-FORMAT: attribute // no pointers to BoundsRanges should be ever used
```

Comment:
If this is required would it make sense to add a private `operator&` to prevent someone taking a pointer to the object?

Reply:
Makes sense, will add as a separate PR later, thanks!